src/base/bittorrent/sessionstatus.h

```diff
@@ -29,50 +29,43 @@
 #ifndef BITTORRENT_SESSIONSTATUS_H
 #define BITTORRENT_SESSIONSTATUS_H
 
-#include <libtorrent/session_status.hpp>
 #include <QtGlobal>
 
 namespace BitTorrent
 {
-    class SessionStatus
+    struct SessionStatus
     {
-    public:
-        SessionStatus(const libtorrent::session_status &nativeStatus);
```

Comment:
eh? Is it wise decision? Libtorrent 1.0 will go, but hardly readable code will stay with us forever.